app/models/vote.rb

```diff
@@ -1,14 +1,2 @@
 class Vote < ActiveRecord::Base
```

Comment:
There is a rake task to migrate old votes into new. The Vote model class is required for that task. If this class is removed we might have to write raw SQL inside the task to access the old votes data table.